src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/ReadDescriptorsValidation.feature

```diff
@@ -104,7 +104,7 @@ Feature: Read a Descriptor
                   """
 
         Scenario: 07 Ensure clients can retrieve a descriptor by requesting through a valid namespace
-             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors?namespace%3Duri%3A%2F%2Fed-fi.org%AbsenceEventCategoryDescriptor"
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors?namespace=uri://ed-fi.org/AbsenceEventCategoryDescriptor#Sick Leave"
```

Comment:
One part of the encoding was wrong - `=` should not have been transformed to `%3D`. But the rest of it needs to be restored, because slash and # need to be encoded in a URL. Unless the C# code is going to handle that implicitly?

For example, here's a valid call to the ODS/API:

```
curl -X 'GET' \
  'https://api.ed-fi.org:443/v7.2/api/data/v3/ed-fi/absenceEventCategoryDescriptors?namespace=uri%3A%2F%2Fed-fi.org%2FAbsenceEventCategoryDescriptor' \
  -H 'accept: application/json' \
  -H 'Use-Snapshot: false' \
  -H 'Authorization: Bearer 30d1c5ac974941d790cca459930f9793'
```

Reply:
Updated